nutils/evaluable.py

```diff
@@ -5708,7 +5333,7 @@ def _compile_to_pyast(funcs, globals, evaluables, new_index, cache, stats, paral
             ndependents[func] = 1
     while stack:
         func = stack.pop()
-        for arg in ((func.init_arg, func.body_arg) if isinstance(func, Loop) else func._Evaluable__args):
+        for arg in func._Evaluable__args:
```

Comment:
Is it correct that the iterable gains the loop's length here? (honestly I'm more surprised by the fact that it wasn't there before, but apparently that worked)

Reply:
Fixed by adding `length` to commit 'add inplace compilation'.

Reply:
Fixed in the referenced commit. Note that in commit 'remove unused Evaluable.evalfs,serialized etc.' the eval args are equal to `init_arg, body_arg`, so we can again use `super().arguments`.